services/galley/src/Galley/API/MLS/Types.hs

```diff
@@ -41,3 +42,9 @@ mkClientMap = foldr addEntry mempty
 
 cmAssocs :: ClientMap -> [(Qualified UserId, (ClientId, KeyPackageRef))]
 cmAssocs cm = Map.assocs cm >>= traverse toList
+
+-- | Inform a handler for 'POST /conversations/list-ids' if the MLS global team
+-- conversation and the MLS self-conversation should be included in the
+-- response.
+data ListGlobalSelfConvs = ListGlobalSelf | DoNotListGlobalSelf
```

Comment:
I think it makes sense to generalise this to the implicit MLS conversations and re-use it for the global team conv as well.